snapd.code-workspace

```diff
@@ -0,0 +1,56 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {
+		"go.testEnvVars": {"LANG": "C.UTF-8"},
+		"go.testFlags": ["-v","-check.v"]
+	},
+	"extensions": {
+		"recommendations": [
+			"golang.go", 
+			"766b.go-outliner", 
+			"jerrygoyal.shortcut-menu-bar",
+			"eamodio.gitlens", 
+			"aaron-bond.better-comments",
+			"vscodevim.vim",
+			"tuttieee.emacs-mcx",
+			"ms-vscode.cpptools-extension-pack",
+		]
+	},
+	"launch": {
+		"version": "0.2.0",
+		"configurations": [
+			{
+				"name": "Test (package)",
+				"type": "go",
+				"request": "launch",
+				"mode": "test",
+				"cwd": "${fileDirname}",
+				"output": "/tmp/go-build${fileBasenameNoExtension}/${relativeFile}/snap.test",
+				"program": "${fileDirname}",
+				"env": {"LANG": "C.UTF-8"},
+				"args": ["-test.timeout=10m0s"],
+			},
+			{
+				"name": "Test verbose (package)",
+				"type": "go",
+				"request": "launch",
+				"mode": "test",
+				"cwd": "${fileDirname}",
+				"output": "/tmp/go-build${fileBasenameNoExtension}/${relativeFile}/snap.test",
+				"program": "${fileDirname}",
+				"env": {
+					"LANG": "C.UTF-8", 
+					//--------------------------------------------------------------------------------
+					"SNAPD_DEBUG": "1"}, 
+					// Some tests that capture STD might fail because of the extra prints issued
```

Comment:
I am a bit unsure how we expect people to get to this comment if they hit the problem or even more before they hit the problem?